iamaccessgroupsv2/iam_access_groups_v2_integration_test.go

```diff
@@ -43,15 +39,22 @@ var _ = Describe("IAM Access Groups - Integration Tests", func() {
 		config                 map[string]string
 		configLoaded           bool = false
 
-		testAccountID        string
-		testGroupName        string = "SDK Test Group - Golang"
-		testGroupDescription string = "This group is used for integration test purposes. It can be deleted at any time."
-		testGroupEtag        string
-		testGroupID          string
-		testUserID           string = "IBMid-" + strconv.Itoa(rand.Intn(100000))
-		testClaimRuleID      string
+		testAccountID	string
+		testGroupName	string = "SDK Test Group - Golang"
+		testGroupDescription	string = "This group is used for integration test purposes. It can be deleted at any time."
+		testGroupEtag	string
+		testGroupID	string
+		testUserID	string = "IBMid-" + strconv.Itoa(rand.Intn(100000))
+		testClaimRuleID	string
 		testClaimRuleEtag    string
 		testAccountSettings  *iamaccessgroupsv2.AccountSettings
+		testPolicyTemplateID	string
+		testTemplateID	string
+		testTemplateEtag	string
+		testLatestVersionETag	string
+		testAccountGroupID	string
+		testAssignmentID	string
+		testAssignmentEtag	string
```

Comment:
Please re-format this to follow the previous style (types on the same column).
`go fmt iamaccessgroupsv2/*.go`